plugin/core/open.py

```diff
@@ -15,6 +15,19 @@
 opening_files = {}  # type: Dict[str, Tuple[Promise[Optional[sublime.View]], ResolveFunc[Optional[sublime.View]]]]
 
 
+def _return_existing_view(flags: int, open_file_group: int, active_group: int, specified_group: int) -> bool:
+    if specified_group > -1:
+        return False
```

Comment:
What if the file is already open in the specified group? Then it should return `True` I guess. That is what the former `opens_in_desired_group` variable did.